backend/app/controllers/api/PagesController.scala

```diff
@@ -112,10 +120,24 @@ class PagesController(val controllerComponents: AuthControllerComponents, manife
     }
   }
 
-  def findInDocument(uri: Uri, fq: String) = ApiAction.attempt {
+  def findInDocument(uri: Uri, fq: String) = ApiAction.attempt { req =>
```

Comment:
As discussed IRL it would be neat if we could use [SSE](https://www.w3schools.com/html/html5_serversentevents.asp) here.

Reply:
Yeah this would be really cool! I think I'll press forward wiring up the main search highlights first and come back to this one, since page viewer work has dragged on so long

Reply:
added to this checklist: https://trello.com/c/ycfzJavQ/1986-page-viewer-navigate-between-individual-highlights